packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderFinal/en.ftl

```diff
@@ -0,0 +1,5 @@
+verificationReminderFinal-subject = Final reminder to confirm your account
+verificationReminderFinal-description = A couple of weeks ago you created a  { -product-firefox-account }, but never confirmed it. For your security, we will delete the account if not verified in the next 24 hours.
+confirm-account = Confirm account
+confirm-account-plaintext = { confirm-account }:
+verificationReminderFinal-action = Confirm account
```

Comment:
I don't think you need this since `confirm-account` is essentially the same string.